catalog/admin.py

```diff
@@ -1,3 +1,14 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Book, BookInstance, Author, Language, Genre
+
+
+
+# The models are registered here so that they can be populated by admin site
+# All models that are registered here are accessible through admin site
+
+admin.site.register(Book)
+admin.site.register(Author)
+admin.site.register(BookInstance)
+admin.site.register(Genre)
+admin.site.register(Language)
```

Comment:
LGTM, just add EOL in this file.

Reply:
I think this file is from the first commit. This issue is already fixed in the files after the commit.